android/build.gradle

```diff
@@ -50,6 +50,11 @@ repositories {
 }
 
 dependencies {
-    //noinspection GradleDynamicVersion
-    implementation 'com.facebook.react:react-native:+'
+    if (project == rootProject) {
+        // If this is the root project (e.g. Jitpack), specify a version
+        implementation 'com.facebook.react:react-native:0.57.5'
```

Comment:
Specifying the version here since `package.json` does not give a specific version.